packages/components/inline-adder/src/js/inline-adder.controller.js

```diff
@@ -16,6 +16,11 @@ export default class {
     this.forms = [true];
     this.isDisabled = [false];
 
+    this.cnt = 0;
+    if (typeof this.rowsLimit === 'undefined') {
```

Comment:
**nitpick:** `typeof` always returns the `string`. So this expression will always evaluates to `false`.